### PR TITLE
Snowmap pr

### PR DESCRIPTION
**Pull request by ghost**
Right, let's see what needs changing

Previous PRs #12994

### CODE REVIEW

**Comment by ghost** [comment]
Oh

Holy fuck that is a lot of stuff to fix

**Comment by SkowronX** [comment]
lol

**Comment by Intigracy** [comment]
So it's going to be one gigantic PR that breaks tons of stuff again then.

Does nobody ever learn?

**Comment by ghost** [comment]
No, just seeing what needs fixing en masse, and what can be cut off and made into a seperate PR

Think of this like a big checklist

**Comment by Zth--** [comment]
whennnn

**Comment by Durgen** [comment]
can we see some screenshots?

File: code/game/objects/closets/walllocker.dm [hunk 1]
```diff
@@ -54,6 +54,27 @@
 	pixel_x = WORLD_ICON_SIZE
 	dir = EAST
 
+/obj/structure/closet/walllocker/emerglocker/snow
+	spawnitems = list(/obj/item/weapon/storage/toolbox/emergency/snow)
```
Comment (F-OS) [inline code/game/objects/closets/walllocker.dm:58]: Why does this one have snow toolboxes and the regular closet have regular toolboxes?

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm [hunk 2]
```diff
@@ -72,6 +75,19 @@
...
+	desc = "It's a decrepit storage unit containing emergency supplies."
+	icon_state = "coldemerg"
+	icon_closed = "coldemerg"
+	icon_opened = "coldemergopen"
+	default_stuff = 0
+
+/obj/structure/closet/emcloset/cold/New()
+	..()
+	for(var/i = 1 to rand(3,4))
+		new /obj/item/weapon/storage/toolbox/emergency(src)
```
Comment (F-OS) [inline code/game/objects/structures/crates_lockers/closets/utility_closets.dm:88]: Again, why.

**Comment by F-OS** [comment]
Alright, played for a bit and spotted some issues.

* The glaciers can sometimes get ohshit textures.

* The map could do with some wolf spawns i think.

* The apc's in the medical and EVA checkpoints on 2 of the taxi stations have no cells in them.

File: code/game/objects/structures/crates_lockers/closets/utility_closets.dm (hunk 2 above)
Comment (kol1th) [reply code/game/objects/structures/crates_lockers/closets/utility_closets.dm:88]: Celt works in mysterious ways.

**Comment by kol1th** [comment]
Sounds right. Those are the last three issues I could find when we were testing it before merge and such. Now to just fix conflicts and those couple things, and improve the general station and it should be **GOOD TO GO**

**Comment by F-OS** [comment]
stale?

**Comment by Zth--** [comment]
Can someone list what's inside this PR? I'd like to help make this possible but I'm not sure where should I start

**Comment by kol1th** [comment]
Shadows
Turf reworks
Radial generator fixing
Mobs (dragonbro, mostly done)
Coats(done and in)
Various other scattered shitcode from celt
Lake generation
Dungeon generation
Custom asteroid stuff


That's just off the top of my head

**Comment by kol1th** [comment]
Oh, and the map itself should be scrapped and re-mapped. There's no functioning disposals system and no engine other than solars. And there are pipes running through the dirt in the snow.

**Comment by ghost** [comment]
Oh yeah, we added a system that allowed for asteroid tiles to have what floor they spawn when destroyed set, and converted ores to an overlay so you can have custom asteroid sprites

**Comment by F-OS** [comment]
Alright if nobody objects to this gonna fork snowmap and try to fix the issues then make a new pr. Since this one seems to be stale.

**Comment by ghost** [comment]
It's literally a checklist and so people can do just that, so go ahead.

Currently, the effort to break it up into bits is kinda hampered, as #14168 is stuck in no-merge hell

**Comment by Anton-Kr** [comment]
When

**Comment by PJB3005** [comment]
Lol

**Comment by ghost** [comment]
The fuck?

It's open so people know where to find the stuff. This ain't dead

**Comment by PromethTK11** [comment]
snowmap when

**Comment by ghost** [comment]
New PR over at https://github.com/vgstation-coders/vgstation13/pull/14794

**Comment by PJB3005** [comment]
Then close it?

**Pull request closed without merging**